src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALL_AMOUNT = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < BALL_AMOUNT; i++) {
+            String randomColor = lottery.getRandomBall().getColor();
+            int randomNumber = lottery.getRandomBall().getNumber();
+            System.out.println("Ball number " + randomNumber + " " + randomColor);
+        }
```

Comment:
Your task is to create 3 balls. Previously you made it but there were comments to plural Lottery creation.
Now you fixed issue with lottery, but you creates balls 6 times (2 times per one loop iteration). Why???
```suggestion
        for (int i = 0; i < BALL_AMOUNT; i++) {
            System.out.println(lottery.getRandomBall());
        }
```